docs/process.md

```diff
@@ -84,12 +91,16 @@ What should reviewers be checking for?
 
 ### Stage 4: Social media
 
-Once published, the writer creates social media posts and shares links on the “DCR Social Media” channel on Matrix. This gives the wider community an opportunity to engage with the article, adding comments, likes and retweets etc.
+Once published, the writer creates social media posts and shares links on the “DCR Social Media” and "DCR Writers" channels on Matrix. This gives the wider community an opportunity to engage with the article, adding comments, likes and retweets etc.
 
 What social media content should writers create?
 
-* Twitter — Initial tweet from writers' Twitter account, maybe do this as a thread and put the link on the last post. They could also schedule a tweet a few days later, perhaps with a poll attached to increase engagement.
+* Twitter — Initial tweet from writers' Twitter account, this could be done as a single post or thread. Don't forget to include the link.
 * Reddit — Post article excerpt and link to Decred’s reddit page to get more interaction on this platform
 * Bison Relay — Post the full article to the Bison Relay News feed.
 * (Optional) Facebook, Instagram, and TikTok — have generally been low engagement platforms for Decred, but this could also be an opportunity to increase engagement on these platforms
 * Comments — Where applicable, start the discussion in the comment section (Cypherpunk Times, Bison Relay, Reddit). This could be a good opportunity to ask questions, e.g. What would you do …?
+
+### Optional: TwitterSpace / Livestream
+
+To increase article engagement the write has the option to create a TwitterSpace / Livestream to discuss the article with the community. This is likely to get a lot more eyes on the article and encourage wider engagement from the community.
```

Comment:
`write` -> `writer`. Also `Twitter Space` (split the words).

Reply:
Done 👍